src/server/enrollment-system.js

```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
           addressTypeCode: 'P',  // TODO(awong): this code is from VHA Standard Data Service (ADRDEV01) Address Type List P==Permanent. Determine if we need it.
         },
         emails: {
-          email: veteran.email,
+          email: [
+            {
+              address: veteran.email,
```

Comment:
From https://github.com/department-of-veterans-affairs/healthcare-application/blob/master/src/server/enrollment-system.js#L991-L992 (and the doc that comes from) it looks like the key is just `address`. I may be misunderstanding how this data is supposed to be built up, though.